company/team/locations.md

```diff
@@ -27,4 +30,27 @@ To add your own location:
         }
     }
     ```
-1. Send a pull request to update the file!
+1. Copy the new JSON from the right side of the page with your updated info included, and use it to generate a PR for [this file](https://github.com/sourcegraph/about/blob/main/company/team/locations.geojson).
+
+
+### Option 2
+1. Use [this site](https://www.latlong.net/) to get your city's latitude and longitude coordinates. 
+1. Edit the following bit of JSON to include your name and coordinates (be sure to put longitude first):
+    ```
+      {
+        "type": "Feature",
+        "properties": {
+            "name": "<YOUR NAME>"
+        },
+        "geometry": {
+            "type": "Point",
+            "coordinates": [
+                <LONGITUDE>,
+                <LATITUDE>
+            ]
+        }
+    }
+    ```
+1. Edit [this file](https://github.com/sourcegraph/about/blob/main/company/team/locations.geojson). Paste in your JSON above the final two lines of the file, leaving the `] }` below what you pasted in.
```

Comment:
Love that you added the link to the file! I did the same in a separate branch but pulled it to avoid conflicts with your more complete edits. Thanks for doing this!

Reply:
Oh, I'm sorry, I didn't realize! Thank you for taking a look. :)

Reply:
Not at all, we basically made the edit at the same time. Great minds.